pyaedt/modeler/circuits/PrimitivesCircuit.py

```diff
@@ -102,6 +103,16 @@ def model_units(self):
         """Model units."""
         return self._modeler.model_units
 
+    @property
+    def schematic_units(self):
+        """Model units."""
+        return self._modeler.schematic_units
+
+    @schematic_units.setter
+    def schematic_units(self, value):
+        """Schematic units. Defaults are ``"meter"``. Options are ``"mm"``, ``"mil"``, ``"cm"``."""
```

Comment:
```suggestion
        """Schematic units. Options are ``"mm"``, ``"mil"``, ``"cm"``. The default is ``"meter"``."""
```
I'm confused because the options listed do not include ``''meter"``.